lib/src/collect.dart

```diff
@@ -11,6 +11,7 @@ import 'util.dart';
 import 'hitmap.dart';
 
 const _retryInterval = Duration(milliseconds: 200);
+const _debugTokenPositions = false;
```

Comment:
How does one use this? Should we use `bool.fromEnvironment` or simiar?

Reply:
Done